src/site/_collections/hooks/utils.js

```diff
@@ -20,10 +20,41 @@
 
 const addPagination = require('../../_utils/add-pagination');
 const filterByLang = require('../../_filters/filter-by-lang');
+const {defaultLocale} = require('../../_data/site');
+const {i18n} = require('../../_filters/i18n');
 
 /**
- * @param {AuthorsItem[]|TagsItem[]} items
- * @return {TODO[]}
+ * @param {VirtualCollectionItem[]} items
+ * @param {string} path
+ * @param {string} [lang]
+ * @return {VirtualCollectionExpandedItem[]}
+ */
+const addFields = (items, path, lang = defaultLocale) => {
```

Comment:
why do you need to translate the title and description here as opposed to in the njk templates? Is this because of the cards?

Reply:
Yes it is. That's the primary use, though I suspect it makes it's easier this way to set the title of paginated pages from these virtual collections. I'll add a comment though.